Documentation/ceph-object-store-crd.md

```diff
@@ -119,3 +132,36 @@ execution attack.
 Rook will not overwrite an existing `mime.types` ConfigMap so that user modifications will not be
 destroyed. If the object store is destroyed and recreated, the ConfigMap will also be destroyed and
 created anew.
+
+## Health settings
+
+Rook-Ceph will be default monitor the state of the object store endpoints.
+The following CRD settings are available:
+
+* `healthCheck`: main object store health monitoring section
+
+Here is a complete example:
+
+```yaml
+healthCheck:
+  endpoint:
+    enabled: true
+    interval: 60s
+```
+
+The endpoint health check procedure is the following:
```

Comment:
For user documentation, how about something higher level? A short description would be enough to explain we're creating a bucket, creating a user, and testing a PUT and GET. They don't need to know all the implementation details.